ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -211,9 +210,9 @@ open class TableViewCell: UITableViewCell, TokenizedControlInternal {
     ///   - isInSelectionMode: Boolean describing if the cell is in multi-selection mode which shows/hides a checkmark image on the leading edge
     /// - Returns: a value representing the calculated height of the cell
     @objc public class func height(tokens: TableViewCellTokens = .init(),
-                                   title: String,
-                                   subtitle: String = "",
-                                   footer: String = "",
+                                   title: Label,
```

Comment:
I see that we're expecting a `Label` view to be passed in here, but I'm not certain that's the best approach. This function (AFAIK) is meant to be a shortcut for helping clients estimate the height of a cell for UITableView layout, so it needs to be a fast lookup, and creating views kinda erases that performance.

Ideally I'd like this API to take in exactly what is needed for measurement, and nothing else. Is there any way we can extract the info we need from the Label to pass in?

Reply:
Yes! I ended up removing label, and adding in fonts. Ideally we'd want to get them from the tokens, but labels can now have separate font values from the tokens with the rich text support.